frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -1,30 +1,114 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
 
 const styles = {
+  //buttons
   button: {
-    width: 180,
-    marginBottom: 10,
-    marginLeft: 20,
-    marginRight: 20,
-    textAlign: "center",
+    width: 155,
+    textAlign: "left",
     borderRadius: 4,
     padding: 6,
-    border: "2px solid #009fae"
+    border: "2px solid #00565E",
+    cursor: "pointer"
+  },
+  buttonSelectedBackground: {
+    backgroundColor: "#00565E"
+  },
+  buttonReadBackground: {
+    backgroundColor: "#F5F5F5"
+  },
+  buttonUnreadBackground: {
+    backgroundColor: "white"
+  },
+  buttonSelectedText: {
+    color: "#D3FCFF"
+  },
+  buttonUnselectedText: {
+    color: "black"
+  },
+  //li
+  li: {
+    listStyleType: "none"
+  },
+  //subject line
+  subjectSelected: {
+    color: "white"
+  },
+  subjectUnselected: {
+    color: "#00565E"
+  },
+  subjectRead: {
+    fontWeight: "normal"
+  },
+  subjectUnread: {
+    fontWeight: "bold"
   }
 };
 
 class EmailPreview extends Component {
   static propTypes = {
     email: PropTypes.object.isRequired,
-    clickFunction: PropTypes.func.isRequired
+    selectEmail: PropTypes.func.isRequired,
+    isRead: PropTypes.bool.isRequired,
+    isRepliedTo: PropTypes.bool.isRequired,
+    isSelected: PropTypes.bool.isRequired
   };
+
   render() {
+    //READ/UNREAD CHECK
+    //defaults, or if unread
+    let buttonBackgroundColor = styles.buttonUnreadBackground;
```

Comment:
These can also be moved into the return logic.

Reply:
fixed in previous push